src/cljs/rems/application.cljs

```diff
@@ -304,7 +304,7 @@
                [fields/field (assoc fld
                                     :on-change #(rf/dispatch [::set-field-value (:field/id fld) %])
                                     :on-set-attachment #(rf/dispatch [::save-attachment (:field/id fld) %1 %2])
-                                    :on-remove-attachment #(rf/dispatch [::remove-attachment (:field/id fld) %1 %2])
+                                    :on-remove-attachment #(rf/dispatch [::remove-attachment (:field/id fld) %])
```

Comment:
I think those %1 and %2 were a copy paste error from on-set-attachment, since ::remove-attachment handler does not use that many fields.